test/XRP/fakes/fake-xrp-client.ts

```diff
@@ -17,6 +17,7 @@ class FakeXrpClient implements XrpClientDecorator {
     public paymentHistoryValue: Result<Array<XrpTransaction>>,
     public getPaymentValue: Result<XrpTransaction>,
     public enableDepositAuthValue: Result<TransactionResult>,
+    public authorizeSendingAccountValue: Result<TransactionResult>,
```

Comment:
I feel like we could get away with something like a `public successfulTransactionResultValue: Result<TransactionResult>` and use it everywhere that we're just looking for a stock successful transaction result, to keep the constructor a little smaller.  Not a big deal though, I think this is the last functionality we'll be adding here for a little while.

Reply:
i thought about this, since i'm gonna add in `unauthorize` as well - let me give it a shot and see how it goes

Reply:
i decided to wait on this one - lemme know if you think i should do it in this PR!

Reply:
nah that's fine, maybe just do it for unauthorize?

Reply:
izzaktly